bundles.rst

```diff
@@ -22,12 +22,17 @@ file::
     return [
         // 'all' means that the bundle is enabled for any Symfony environment
         Symfony\Bundle\FrameworkBundle\FrameworkBundle::class => ['all' => true],
-        Symfony\Bundle\SecurityBundle\SecurityBundle::class => ['all' => true],
-        Symfony\Bundle\TwigBundle\TwigBundle::class => ['all' => true],
-        Symfony\Bundle\MonologBundle\MonologBundle::class => ['all' => true],
         Doctrine\Bundle\DoctrineBundle\DoctrineBundle::class => ['all' => true],
+        Doctrine\Bundle\MigrationsBundle\DoctrineMigrationsBundle::class => ['all' => true],
+        // this bundle is enabled only in 'dev'
+        Symfony\Bundle\DebugBundle\DebugBundle::class => ['dev' => true],
+        Symfony\Bundle\TwigBundle\TwigBundle::class => ['all' => true],
         // this bundle is enabled only in 'dev' and 'test', so you can't use it in 'prod'
         Symfony\Bundle\WebProfilerBundle\WebProfilerBundle::class => ['dev' => true, 'test' => true],
+        Twig\Extra\TwigExtraBundle\TwigExtraBundle::class => ['all' => true],
+        Symfony\Bundle\SecurityBundle\SecurityBundle::class => ['all' => true],
+        Symfony\Bundle\MonologBundle\MonologBundle::class => ['all' => true],
+        Symfony\Bundle\MakerBundle\MakerBundle::class => ['dev' => true],
```

Comment:
Lets sort the alpabetically

Reply:
My goal there was to present this file as it is after a standard install (—webapp) to avoid confusion to newcomers